bgpd/bgp_table.h

```diff
@@ -42,6 +42,31 @@ struct bgp_table {
 	uint64_t version;
 };
 
+enum bgp_path_selection_reason {
+	bgp_path_selection_none,
+	bgp_path_selection_first,
+	bgp_path_selection_evpn_sticky_mac,
+	bgp_path_selection_evpn_seq,
+	bgp_path_selection_evpn_lower_ip,
+	bgp_path_selection_weight,
+	bgp_path_selection_local_pref,
+	bgp_path_selection_local_route,
+	bgp_path_selection_confed_as_path,
+	bgp_path_selection_as_path,
+	bgp_path_selection_origin,
+	bgp_path_selection_med,
+	bgp_path_selection_peer,
+	bgp_path_selection_confed,
+	bgp_path_selection_igp_metric,
+	bgp_path_selection_older,
+	bgp_path_selection_router_id,
+	bgp_path_selection_cluster_length,
+	bgp_path_selection_stale,
```

Comment:
Few of these reasons can be sub divided into two. For example, bgp_path_selection_confed, bgp_path_selection_older, bgp_path_selection_stale etc.